### ISSUE TITLE
Optional SRTP with duplicated media line

### ISSUE THREAD

**Issue by pjsipbot** [pjsip/pjproject#1267]
_2011-05-19 08:47:37: @bennylp_ _created the issue on trac ticket [1267](https://trac.pjsip.org/repos/ticket/1267)_

As this SRTP mode is handled by high level library, i.e: PJSUA-LIB, and ticket #1185 causes a lot of modification in handling media transport in PJSUA-LIB, this SRTP mode is 'accidentally' stripped out. 

-*Note**: test scenario `mod_call.py scripts-call/150_srtp_2_3.py` is currently disabled as it depends on optional SRTP with duplicated media line. So don't forget to re-enable the scenario (in `runall.py`) once this ticket is done.
___
_2011-07-11 04:29:09: @bennylp_ _changed **milestone** from **release-2.0-alpha** to **release-2.x**_

___
_2011-08-19 13:10:54: @nanangizz_ _edited the issue description_

___
_2011-08-19 13:43:22: @nanangizz_ _edited the issue description_

___
_2013-10-05 04:17:13: @bennylp_ _changed **status** from **new** to **closed**_

___
_2013-10-05 04:17:13: @bennylp_ _set **resolution** to **wontfix**_
